diff --git a/tmenu.c b/tmenu.c
index 26fba31..401f9ab 100644
--- a/tmenu.c
+++ b/tmenu.c
@@ -40,6 +40,7 @@ static int top = TOP;
 static int ln = LINES;
 static const char *pr = PROMPT;
 static unsigned long col[4];
+static unsigned int border_width = 2;
 
 static unsigned long xc(const char *s) {
 	XColor c;
@@ -247,9 +248,11 @@ int main(int argc, char *argv[]) {
 	wa.override_redirect = 1;
 	wa.background_pixel = col[1];
 	wa.event_mask = ExposureMask | KeyPressMask | StructureNotifyMask;
-	w = XCreateWindow(dp, root, 0, top ? 0 : sh - wh, ww, wh, 0,
+	w = XCreateWindow(dp, root, 0, top ? 0 : sh - wh, ww, wh, border_width,
 		DefaultDepth(dp, scr), CopyFromParent, DefaultVisual(dp, scr),
 		CWOverrideRedirect | CWBackPixel | CWEventMask, &wa);
+	if (border_width)
+		XSetWindowBorder(dp, w, xc(BGSEL));
 	pm = XCreatePixmap(dp, w, ww, wh, DefaultDepth(dp, scr));
 	if(pm == None) return 1;
 	gc = XCreateGC(dp, w, 0, NULL);
